### ISSUE TITLE
Search

### ISSUE THREAD

**Issue by sawyerh** [siteleaf/learn.siteleaf.com#5]

**Comment by sskylar** [comment]
Here's the embed code needed for search:

```html
<input type="text" class="st-default-search-input">

<script type="text/javascript">
(function(w,d,t,u,n,s,e){w['SwiftypeObject']=n;w[n]=w[n]||function(){
(w[n].q=w[n].q||[]).push(arguments);};s=d.createElement(t);
e=d.getElementsByTagName(t)[0];s.async=1;s.src=u;e.parentNode.insertBefore(s,e);
})(window,document,'script','//s.swiftypecdn.com/install/v2/st.js','_st');

_st('install','WgemsSzpiSJN3aaHTqYn','2.0.0');
</script>
```

Tip: You can nav content from being index using:

```
  <p data-swiftype-index='false'>
    Content in this paragraph tag will be excluded from the search index!
  </p>
```

https://swiftype.com/documentation/content_inclusion_exclusion

You can also use meta tags for more structured indexing:
https://swiftype.com/documentation/meta_tags

e.g. https://github.com/siteleaf/siteleaf.com/blob/master/_layouts/default.html#L7

**Comment by eturk** [comment]
Closed with #30.
